en/dronecan/README.md

```diff
@@ -100,6 +102,18 @@ To enable the DroneCAN driver, set the [UAVCAN_ENABLE](../advanced_config/parame
 
 `2` or `3` are recommended, if DNA is supported.
 
+### DroneCan Publications
+
+DroneCAN does not publish all information that _might_ be needed, in order to avoid spamming the CAN bus.
+Publishing of specific information is enabled it using the associated [UAVCAN parameter](../advanced_config/parameter_reference.md#uavcan).
+These can be recognised from the prefix `UAVCAN_PUB_`.
+
+The set of publications that you can enable are (in PX4 v1.14):
+
+- [UAVCAN_PUB_RTCM](../advanced_config/parameter_reference.md#UAVCAN_PUB_RTCM) ([RTCMStream](https://dronecan.github.io/Specification/7._List_of_standard_data_types/#rtcmstream)): Enable when using a CAN-connected [RTK GPS](../gps_compass/rtk_gps.md) as the Rover unit (it ensures that RTCM information from the base unit is returned to the rover).
```

Comment:
Is this correct? I ask because this is NOT set by the ARK GPS instructions: https://docs.px4.io/main/en/dronecan/ark_rtk_gps.html

Is it that if you set the moving base publication this implicitly also sends this data?

Reply:
I don't know.

Reply:
@AlexKlimaj Do you know?

Reply:
Technically CANNODE_PUB_MBD and UAVCAN_PUB_RTCM both send RTCM "gps_inject_data" messages. They correspond to different dronecan messages though to allow a static base and moving base setup simultaneously.

Reply:
@AlexKlimaj Thanks. The problem with this answer is that I don't understand the architecture, so no idea if this is a significant point or a "by the way". 

My guess here is that CANNODE_PUB_ reflects a publication from a peripheral, while  UAVCAN_PUB_ reflects a publication from PX4. So you're saying that 
- CANNODE_PUB_MBD causes the moving base GPS unit to send RTCM messages onto the bus (to the moving base)
- CANNODE_SUB_MBD tells the rover that it should subscribe to RTCM messages on the bus (from the moving base)
- UAVCAN_PUB_RTCM tells PX4 it should publish RTCM messages to the bus for a connect static base unit (which it gets from QGC).

Is that right?

Reply:
Yes CANNODE_PUB_ is to publish from the node and  UAVCAN_PUB_ is to publish from the flight controller. Same for the _SUB param for subscribing. 

- CANNODE_PUB_MBD causes the moving base GPS unit to send RTCM messages onto the bus (to the rover) (movingbaselinedata dronecan message)
- CANNODE_SUB_MBD tells the rover that it should subscribe to RTCM messages on the bus (from the moving base) (movingbaselinedata dronecan message)
- UAVCAN_PUB_RTCM tells PX4 it should publish RTCM messages to the bus for a connect static base unit (which it gets from QGC). (rtcmstream dronecan message)
- CANNODE_SUB_RTCM tells the rover that it should subscribe to RTCM messages on the bus (from the moving base) (rtcmstream dronecan message)

rtcmstream is the old uavcan dsdl message. movingbaselinedata was added to dronecan under ardupilot which increased the message size. Having both messages actually allows for a fixed base and moving base with corrections streamed over the same bus.